lib/util/removeUnnecessaryParens.js

```diff
@@ -37,7 +37,12 @@ function removeUnnecessaryParensSearch(node) {
     return node;
   }
   else if (Node.Type.isUnaryMinus(node)) {
-    const content = node.args[0];
+    let content = node.args[0];
+
+    if (Node.Type.isParenthesis(content)) {
+      content = removeUnnecessaryParensInParenthesisNode(content)
+    }
```

Comment:
here the assumption that I'm working with is that we can remove unnecessary parentheses from the arg of a unary minus. I don't see why we couldn't?